### PR TITLE
Update tutorial material

### PR DESCRIPTION
**Pull request by elemoine**
This PR updates the tutorial material. Mainly involves passing the path to the working directory to the DVC commands (using the `-w` option), and making sure the requirements install properly. I did not verify the whole workshop, but things seem to work.

### CODE REVIEW

**Comment by elemoine** [comment]
@SarahDiotGirard please tell me if I can merge this. Thanks :-)

File: README.md [hunk 1]
```diff
@@ -107,106 +108,116 @@ Keep in mind that:
...
-**DVC pipeline steps**: DVC command applied on generated Python 3 scripts
+For each Jupyter notebook a Python parameterizable and executable script is generated.
+It is the way to version code and be able to automatize its run.
 
+Pipelines are composed of DVC steps. Those steps can be generated directly from the
+Jupyter notebook based on parameters described in the Docstring. (notebook -> python
+script -> DVC command)
 
-#### Applying the process
+Each time a DVC step is run a DVC meta file (`[normalize_notebook_name].dvc`) is
```
Comment (SdgJlbl) [inline README.md:120]: s/normalize/normalized

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 SHELL := /bin/bash
 develop:
+	pip install cython
```
[the remark this answers is not in the document]
Comment (elemoine) [reply Makefile:3]: Yes we do. Some package requires that cython is installed before installation.

File: README.md [hunk 3]
```diff
@@ -1,56 +1,55 @@
...
 - git
-- python3
-
+- python
 
 ## Tools Overview
 
-DVC: an open-source tool for data science and machine learning projects. Use to version, share and reproduce.
-
-MLflow tracking: API and UI to log and visualize metrics obtained during experiments.
```
[the remark this answers is not in the document]
Comment (elemoine) [reply README.md:29]: Yes. I removed it from the README initially, and then realized that it was used. MLFlow UI is used IIRC.

File: README.md [hunk 4]
```diff
@@ -1,56 +1,55 @@
...
 ## Standard Versioning Process Establishment
 
 **Goal:** find a way to version code, data and pipelines.
 
-#### Existing project
+### Initial project
 
-Starting from an existing project composed of Python 3 module(s) and a set of **Jupyter notebooks**,
-we want to create an automated pipeline in order to version, share and reproduce experiments.
+Starting from an existing project composed of Python module(s) and a set of Jupyter
```
[the remark this answers is not in the document]
Comment (elemoine) [reply README.md:50]: This part of the tutorial is based on an actual project provided by the tutorial. So I changed the sentence to "Starting from an existing project composed of multiple Python modules and a set of Jupyter notebooks…".

File: README.md [hunk 5]
```diff
@@ -107,106 +108,116 @@ Keep in mind that:
     │── setup.py
 
 
-**Notebooks converted into configurable Python 3 scripts**: obtained by **Jupyter notebook** conversion.
+### Applying the process
 
-**DVC pipeline steps**: DVC command applied on generated Python 3 scripts
+For each Jupyter notebook a Python parameterizable and executable script is generated.
+It is the way to version code and be able to automatize its run.
```
[the remark this answers is not in the document]
Comment (elemoine) [reply README.md:114]: My goal was not to change the phrasing, but fix what needed to be fixed and the formatting. But I'm obviously happy to accept your good suggestions :-)

Done.

**Pull request merged**